news/_posts/2021-12-14-prestashop-beyond-1-7.md

```diff
@@ -0,0 +1,101 @@
+
+As we said last year, we think that one of the key challenges for achieving PrestaShop's technical transformation is about finding the right pace, so that it can be delivered both progressively and predictably. We believe that the best answer for that is to steer the project towards an iterative cycle of **smaller, more focused, more frequent releases** — each building on the work laid out by the previous one.
+
+## Introducing a new release roadmap
+
+The [Semantic Versioning convention](https://build.prestashop.com/news/a-more-semantic-versioning-scheme/) (SemVer) only allows introducing backward-incompatible changes in major versions. This means that in order to boost PrestaShop's technical transformation, major versions need to be published more often than once every five years or more. At the same time, the community needs time to adapt to changes, as they carry a cost for merchants and developers alike. They need visibility to anticipate **what** kind of changes to expect, **when** will they encounter them, and finally, **how long** will their investment last until they need to adapt again.
+
+This requires a delicate balance: change too slowly, and project becomes obsolete; change too quickly, and it becomes too expensive for the community to keep up. With that in mind, here is our proposal.
+
+The new release roadmap will adhere more closely to the SemVer definition: a version is considered "major" when it introduces breaking changes, while minor versions only add features in a retrocompatible way. By following these principles strictly, upgrading from a major version to the minor version following it should be as easy as upgrading to a patch version, as everything is expected to work the same way. In this sense, it also means that a major version doesn't necessarily mean _big_ changes, just _some_ changes.
+
+Starting on our next release, PrestaShop will begin alternating between major versions, focusing on refactoring and technology advancements, and minor versions, emphasizing on new or improved features. This means that **the next PrestaShop release will be a major version**, which will be followed by a minor version, then a major version, then a minor version, and so on.
+
+Our target is to be able to publish a release every 6–9 months. Since every other release will be a major version, the community will be able to expect a new major version every 12 to 18 months.
+
+Now let's talk about support. Currently, patch versions are released at most 6 weeks counting from the moment the maintainers decided that a given bug justifies a patch release. Since patches are only published for the latest version, this effectively means that support of a given PrestaShop minor version generally ends the moment the following minor version is published (e.g. support for 1.7.7.x version ended when 1.7.8.0 was released). This will continue to be true with the new release roadmap.
```

Comment:
Maybe a new section ? So a new h3 above ?

Reply:
This is part of the same argument for the roadmap, following the "what", "when" and "how long". I find that having too many sections disrupts the reading experience.